src/cogent3/core/new_alignment.py

```diff
@@ -187,9 +188,34 @@ def copy(self, sliced: bool = False):
         """returns copy"""
         return self
 
-    def to_rich_dict(self) -> dict: ...
+    def to_rich_dict(self) -> dict[str, str | dict[str, str]]:
+        """returns a json serialisable dict.
+
+        Notes
+        -----
+        This method will slice the underlying sequence to the start and stop values
+
+        Warning
+        -------
+        This method is not intended to provide deserialisation of this object,
```

Comment:
change "deserialisation" to "serialisation" and "it is intended to usage by an eclosing class".

Also, if this is the `SeqDataView` class then it cannot work for deserialisation since that class requires it's `seq` object to be a `SeqsDataABC`.